DeComp/definitions.py

```diff
@@ -376,8 +384,8 @@ class functions do.
     "squashfs": [
                     "_common", "unsquashfs",
                     [
-                        "other_options", "-d", "%(destination)s",
-                        "%(basedir)s/%(source)s"
+                        "other_options", "-f", "-d", "%(destination)s",
+                        "%(source)s"
                     ],
```

Comment:
I don't like this change.   You dropped %(basedir)s/  which is common to all, plus you can use the other_options variable to add the -f flag for your use case without hard coding it..   This change forces the -f flag to everyone.